cmd/node/config/config.toml

```diff
@@ -938,3 +938,6 @@
         Capacity = 1000
         Type = "SizeLRU"
         SizeInBytes = 314572800 #300MB
+
+[FireHoseConfig]
+    Enabled = true
```

Comment:
by default I think this should be false.
Instead of a config parameter I would add a build flag to enable firehose

Reply:
Right, good suggestion. I should've started with flag implementation, not config. 
Refactored